frontend/src/apiMocks/mockData.ts

```diff
@@ -1,8 +1,37 @@
 import { CustomAliasData, RandomAliasData } from "../hooks/api/aliases";
 import { ProfileData } from "../hooks/api/profile";
+import { RuntimeData } from "../hooks/api/runtimeData";
+import { UserData } from "../hooks/api/user";
 
 export const mockIds = ["empty", "onboarding", "some", "full"] as const;
 
+// This is the same for all mock users, at this time:
+export const runtimeData: RuntimeData = {
+  FXA_ORIGIN: "https://example.com",
+  GOOGLE_ANALYTICS_ID: "UA-123456789-0",
+  PREMIUM_PRODUCT_ID: "prod_123456789",
+  PREMIUM_PLANS: {
+    country_code: "nl",
+    plan_country_lang_mapping: {
+      nl: {
+        nl: {
+          id: "price_1JmROfJNcmPzuWtR6od8OfDW",
+          price: "€0,99",
+        },
+      },
+    },
+    premium_countries: ["nl"],
+    premium_available_in_country: true,
+  },
```

Comment:
Not necessarily a risk, but it could indeed be useful to also have a mock account for which Premium is not available, in case we want to test/demo UI in that situation. That's pretty easy to add using the same mechanism as used for other accounts, but let's cross that bridge when we get to it, because it's yet another mock to maintain.